pom.xml

```diff
@@ -24,7 +24,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <openMRSVersion>2.1.1</openMRSVersion>
+        <openMRSVersion>2.1.3-SNAPSHOT</openMRSVersion>
```

Comment:
Looks like 2.1.3 was already released, so this shouldn't point to a snapshot: https://github.com/openmrs/openmrs-core/releases/tag/2.1.3